tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Moved this up one because my vscode was not picking up its rules in my text editor/